docs/components/tasklist/deployment/authentication.md

```diff
@@ -15,17 +15,22 @@ By default, user storage in Elasticsearch is enabled.
 
 In this mode, the user authenticates with a username and password stored in Elasticsearch.
 
-The **username** and **password** for one user may be set in application.yml:
+The **username**, **password** and **roles** for one user may be set in application.yml:
 
 ```
 camunda.tasklist:
   username: anUser
   password: aPassword
+  roles:
+    - OWNER
+    - USER
```

Comment:
in [Operate #1672](https://github.com/camunda-cloud/operate/pull/1762/files) you created `FULL_ACCESS`. Are we removing it? I think it makes sense to remove it, as it is the same as `OWNER`

Reply:
It's just an example to show that you can assign a list of roles. For now you're right it isn't necessary to assign an user to both `OWNER` and `USER`